### PR TITLE
fix: msft365 spells INFO severity as informational

### PR DESCRIPTION
**Pull request by edyesed**
### Background

We received reports that the INFO severity passthrough was not functional due to a spelling mismatch.  cc/ @hbenac10 

### Changes

* 

### Testing

*

### CODE REVIEW

File: rules/microsoft_rules/microsoft_graph_passthrough.py [hunk 1]
```diff
@@ -14,6 +14,8 @@ def dedup(event):
 
 
 def severity(event):
+    if event.get("severity") == "informational":
```
Comment (maxrichie5) [inline rules/microsoft_rules/microsoft_graph_passthrough.py:17]: Want to `lower` this string?

Comment (edyesed) [reply rules/microsoft_rules/microsoft_graph_passthrough.py:17]: eh, may as well 👍 . in a data sample, all values were already lowercased

**Review by maxrichie5 — APPROVED** [review]

**Comment by jpl5280** [comment]
@edyesed - thoughts on updating the code, to where we throw an error or something when this happens? Feels odd we had to learn from a customer

**Comment by edyesed** [comment]
> @edyesed - thoughts on updating the code, to where we throw an error or something when this happens? Feels odd we had to learn from a customer

I concur, we may need to revisit the passthrough types of `severity()` functions and confirm that they always return valid values.

**Pull request merged**